src/pages/payment-methods/_expanded-list.tsx

```diff
@@ -1,20 +1,23 @@
 import React from 'react'
-import PropTypes from 'prop-types'
 import styled, { css } from 'styled-components'
+import { PaymentProps } from './index'
 import { Button } from 'components/form/'
 import { Text } from 'components/elements'
 import { localize } from 'components/localization'
-// import { getCryptoDecimals } from 'common/utility'
-// SVG
 import Chevron from 'images/svg/custom/chevron-thick.svg'
 import PDF from 'images/svg/regulatory/pdf-icon-black.svg'
 
+type ExpandListType = {
+    expanded?: boolean
+    is_expanded?: boolean
```

Comment:
Can we only use `is_expanded` and update the ones that are using `expanded`

Reply:
okay